Assets/Scripts/Models/Character/Character.cs

```diff
@@ -818,8 +799,8 @@ private void Update_DoMovement(float deltaTime)
         // But when we do the pathfinding system, we'll likely
         // switch to something like Manhattan or Chebyshev distance
         float distToTravel = Mathf.Sqrt(
-            Mathf.Pow(CurrTile.X - nextTile.X, 2) +
-            Mathf.Pow(CurrTile.Y - nextTile.Y, 2));
+                                 Mathf.Pow(CurrTile.X - nextTile.X, 2) +
```

Comment:
Just out of curiosity. What is adding so much space before. In MonoDevelop when I press enter after ( it just indents the code 4 space. I also think that having the last ```);``` in a new line might look better, if the indentation is 4 spaces.